recipes/xcube/meta.yaml

```
@@ -0,0 +1,87 @@
+{% set name = "xcube" %}
+{% set version = "0.2.0.dev1" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/dcs4cop/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "2617a04fb38511b9bba629e57c5358c4c2cb2384ec216334dd6f6947b43a80a3"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here.
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+
+requirements:
+  build:
```

This should be `host`
```suggestion
  host:
```